docker-compose.yml

```diff
@@ -55,7 +68,11 @@ services:
     build:
       context: ../tooling-invoker
       dockerfile: Dockerfile.dev
+    depends_on:
+      - startup
+      - tooling-orchestrator
     environment:
+      - TOOLING_WORKER_SLEEP=2
```

Comment:
1 second is way to fast IMHO, so this makes it configurable.

Reply:
Can you move this into a different PR pls. Let's keep this one about the one thing.